hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -593,4 +593,22 @@ class AzkabanHelper {
     }
   }
 
+  /**
+   * Prints a spinning loader
+   * @param times The number of times spin should occur
+   * @param delay The delay in each spin
+   */
+  static void printSpinningLoader(long times, long delay) {
```

Comment:
Can we move the utility functions like these which are being used outside hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban from AzkabanHelper to a new helper class in utils?